### ISSUE TITLE
improve scripts/ develop github actons for updating user_examples and test_models

### ISSUE THREAD

**Issue by k-doering-NOAA** [nmfs-stock-synthesis/workflows#39]
Currently, there are R scripts for updating the models in user-examples and test-models upon each release. These are run locally, but the scripts require some manual changes and take local computing power to run. I feel it takes me longer than it should to run this because I have to refamiliarize myself with the script each time.

Another issue to updating is not well documented, because the script requires some manual changes and doesn't provide good instructions on how to run it. This is not helpful if anyone besides me needs to do the updating.

It may be nicer to fully automate these in github actions. The steps would be:
1. Download the latest SS3 executable
2. Run a streamlined version of the script that doesn't require manual input changes. This would update model input and output files.
3. Delete any files that shouldn't be committed
4. Open a pull request with the changes to the repository for review.

A tag would also need to be added once it is merged in, but this step may need to be left as manual.

**Comment by k-doering-NOAA** [comment]
Another reason to fully automate updating the test results on each release are that the compiler/os is then different, which can lead to some small differences in model results. It would be cleaner to run the model updates on ubuntu since that is the system used to run the stock synthesis regression tests.

**Comment by k-doering-NOAA** [comment]
These scripts are linked in https://github.com/nmfs-stock-synthesis/stock-synthesis/issues/352#issue-1308326395

**Comment by e-gugliotti-NOAA** [comment]
There are now github actions for updating these the user-example models and the test models in the respective repositories. 
[user-example models github action](https://github.com/nmfs-stock-synthesis/user-examples/blob/main/.github/workflows/update-ss3-models.yml)
[test models github action](https://github.com/nmfs-stock-synthesis/test-models/blob/main/.github/workflows/update-ss3-models.yml)
